src/main/msp/msp_protocol.h

```diff
@@ -59,7 +59,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   36 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   37 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
@azolyoung , as @DieHertz says, the version will be increased as part of the BF 3.2 release. The development team does that as part of the release process. So you can use the test `if (semver.gte(CONFIG.apiVersion, "1.37.0")) {`